services_app/src/main/res/values/strings.xml

```diff
@@ -258,8 +258,10 @@
     <string name="admin_password_disabled_click_to_set">No Admin password. Click to enable.</string>
     <string name="admin_password_settings">Admin Password</string>
     <string name="admin_password">Admin Password</string>
-    <string name="change_admin_password">Change admin password</string>
-    <string name="enable_admin_password">Enable user restrictions</string>
+    <string name="user_restrictions">Restrictions</string>
+    <string name="enable_user_restrictions">Enable user restrictions</string>
+    <string name="change_admin_password">Enable Admin Password</string>
```

Comment:
Shouldn't this be "Change" Admin Password? The problem is two strings both have the same value. String names "change_admin_password" and "enable_admin_password" currently have the same value which seems incorrect as each string value should be unique.